src/components/OSCALDiagram.js

```diff
@@ -14,7 +14,7 @@ export default function OSCALDiagram(props) {
     diagramUri = resolveLinkHref(
       props.backMatter,
       link.href,
-      props.parentUrl,
+      props.parentUrl.startsWith(".") ? null : props.parentUrl,
```

Comment:
Oof I really don't know if I like this... I want to find something better than `startsWith(".")` if we do

Reply:
Since we rely on the error checking with `URL()`, this is needed at least for our tests. Otherwise we pass the parent url as `./`, which makes an invalid url. I also think removing it may lead to some wonkiness with the actual `Viewer` but am not sure if we have any sample data to be sure.